templates/demo/app/components/ens.js

```diff
@@ -146,7 +146,7 @@ class ENS extends React.Component {
           </FormGroup>
         </Form>
 
-        <h3>Register subdomain for embark</h3>
+        <h3>Register subdomain for eth</h3>
```

Comment:
We should probably remove the last part in fact, since it can be changed in the config